util/lint_testplan.py

```diff
@@ -6,8 +6,8 @@
 """
 This script lint the various opentitan testplan.hjson
 usage:
-    ./util/lint_testplan.py --schema hw/lint/sival_testplan_schema.hjson \
-                            --dir hw/top_earlgrey/data/ip/
+    python3 ./util/lint_testplan.py --schema hw/lint/sival_testplan_schema.json \
```

Comment:
Is the name change intentional here? I think we might have lost an "h".

Reply:
Good spot! Small oversight here on my part while rebasing and this slipped through.